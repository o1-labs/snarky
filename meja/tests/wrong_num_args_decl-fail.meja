type t('a) = A('a, 'b) : t('a, 'b)
