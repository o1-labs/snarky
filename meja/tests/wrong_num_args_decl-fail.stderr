File "tests/wrong_num_args_decl-fail.meja", line 1, characters 25-34:
Error: The type constructor t expects 1 argument(s)
       but is here applied to 2 argument(s).
